app/models/manageiq/providers/atomic/container_manager.rb

```diff
@@ -13,7 +13,7 @@ def self.ems_type
   end
 
   def self.description
-    @description ||= "Atomic".freeze
+    @description ||= "Atomic Project".freeze
```

Comment:
According to http://www.projectatomic.io, the description should either be `Project Atomic` or plain `Atomic`.  I am leaning towards the plainer name.